dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/react';
+import { from } from '@guardian/source-foundations';
+import { useEffect, useState } from 'react';
+import { getZIndex } from '../lib/getZIndex';
+import { useHasBeenSeen } from '../lib/useHasBeenSeen';
+
+const buttonStyles = css`
+	position: absolute;
+	left: 0;
+	top: 0;
+	${getZIndex('sticky-video-button')};
+`;
+
+const stickyStyles = css`
+	@keyframes fade-in-up {
+		from {
+			transform: translateY(100%);
+			opacity: 0;
+		}
+
+		to {
+			transform: translateY(0%);
+			opacity: 1;
+		}
+	}
+
+	position: fixed;
+	bottom: 20px;
+	width: 300px;
+	height: 169px;
+	${getZIndex('sticky-video')};
+	animation: fade-in-up 1s ease forwards;
+
+	figcaption {
+		display: none;
+	}
+`;
+
+const stickyContainerStyles = (height: number) => css`
+	height: ${height * 0.75}px;
+	position: relative;
+	display: flex;
+	justify-content: flex-end;
+
+	${from.tablet} {
+		height: ${height * 1.5}px;
+	}
+`;
```

Comment:
Bit odd that `height` is never actually used, only derived to 75% and 150%.

Maybe we could start with the the base height and make the tablet and over 200%?

Reply:
good spot, thanks Max